test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
+
+function getUserToken() {
```

Comment:
I considered trying to use a selenium-based session id, but the session id is not sent in the request to the mock server, and using less magic around this felt more maintainable.